utils/https.c

```diff
@@ -31,25 +30,60 @@ int https_logger_release() {
   return 0;
 }
 
-status_t send_https_msg(char const *host, char const *port, char const *api, const char *msg, const char *ssl_seed) {
+static int https_parser_callback(http_parser* parser, const char* at, size_t length) {
```

Comment:
Why didn't you declare the return type as `bool`?

Reply:
The callback has been defined by the http_parser.
see : https://github.com/nodejs/http-parser/blob/d0f3b0548d41acd1d3992f6c36b192620500be58/http_parser.h#L90